provisioning/roles/sp_mini_3_build_go_projects/tasks/main.yml

```diff
@@ -40,42 +40,44 @@
     src: "{{control_plane_dir}}/snowplow-mini-control-plane-api"
     dest: "{{executables_dir}}"
 
-- name: Create caddy source dir
-  become: yes
-  environment:
-    GOPATH: "{{ go_path }}"
-  file:
-    path: $GOPATH/src/github.com/mholt
-    state: directory
+# - name: Create caddy source dir
+#   become: yes
+#   environment:
+#     GOPATH: "{{ go_path }}"
+#   file:
+#     path: $GOPATH/src/github.com/mholt
+#     state: directory
 
-- git:
-    repo: 'https://github.com/mholt/caddy.git'
-    dest: /usr/local/src/caddy
-    version: v0.10.10
+# - git:
+#     repo: 'https://github.com/mholt/caddy.git'
+#     dest: /usr/local/src/caddy
+#     version: v0.10.10
 
-- name: Locate caddy
-  become: yes
-  environment:
-    GOPATH: "{{ go_path }}"
-  command: mv /usr/local/src/caddy $GOPATH/src/github.com/mholt/
+# - name: Locate caddy
+#   become: yes
+#   environment:
+#     GOPATH: "{{ go_path }}"
+#   command: mv /usr/local/src/caddy $GOPATH/src/github.com/mholt/
 
-- name: Get Caddyserver builds
+- name: Get Caddyserver build
   environment:
     PATH: "{{ lookup('env','PATH') }}:{{ go_bin }}"
     GOPATH: "{{ go_path }}"
-  shell: "go get -u github.com/caddyserver/builds"
+  shell: |
+    export GO111MODULE=on
+    go get github.com/mholt/caddy/caddy
 
-- name: Build Caddy
-  environment:
-    PATH: "{{ lookup('env','PATH') }}:{{ go_bin }}"
-    GOPATH: "{{ go_path }}"
-  shell: "cd $GOPATH/src/github.com/mholt/caddy/caddy; go run build.go -goos=linux -goarch=amd64"
+# - name: Build Caddy
+#   environment:
+#     PATH: "{{ lookup('env','PATH') }}:{{ go_bin }}"
+#     GOPATH: "{{ go_path }}"
+#   shell: "cd $GOPATH/src/github.com/mholt/caddy/caddy; go run build.go -goos=linux -goarch=amd64"
```

Comment:
shoudl all those commented lines be removed?

Reply:
Yeah, I'll remove them.